cluster.yml

```diff
@@ -29,16 +29,17 @@
 - hosts: k8s-cluster:etcd
   any_errors_fatal: "{{ any_errors_fatal | default(true) }}"
   roles:
-    - { role: kubespray-defaults}
+    - { role: kubespray-defaults }
     - { role: kubernetes/preinstall, tags: preinstall }
     - { role: "container-engine", tags: "container-engine", when: deploy_container_engine|default(true) }
-    - { role: download, tags: download, when: "not skip_downloads" }
+    - { role: crictl, when: not container_manager == 'docker' }
```

Comment:
Please be consistent:
```
container_manager in ["crio", "containerd"]
```

Reply:
So you'd rather maintain a list of supported managers instead of just the one where it's not possible?

Reply:
I'd rather it be consistent is all. There are 16 conditionals that check `container_manager in` and one bad one of `container_manager != docker` in roles/kubernetes/node/templates/kubelet.env.v1beta1.j2

Reply:
I'll fix that. No problem